pandas/tests/tools/test_to_datetime.py

```diff
@@ -291,6 +291,7 @@ def test_to_datetime_format_microsecond(self, cache):
                 marks=pytest.mark.xfail(
                     locale.getlocale()[0] in ("zh_CN", "it_IT"),
                     reason="fail on a CI build with LC_ALL=zh_CN.utf8/it_IT.utf8",
+                    strict=False,
```

Comment:
can we just remove these xfails entirely?

Reply:
No, these tests are executed on other workers, and they pass. So they are important :)

Reply:
Can we be make the xfail condition more specific?  `strict=False` is _really_ undesirable

Reply:
It's not clear to me why `strict=False` is needed i.e. why does it sometimes pass? Is it because the locale setting isn't threadsafe? If so, these tests can be decorated with `@pytest.mark.single_cpu`

Reply:
It passes when the local computer's locale is not italian or chinese. 

Since most workers in the github actions workflows have a standard en_US locale, the tests pass. This is totally unrelated to threading.

Reply:
Ah okay so this is for local testing. Okay makes sense for this to be `strict=False` then. Could you maybe write a small comment in the code to that effect.

Reply:
ok

Reply:
This is less specific IMO. See below : on test_to_time you use a `fails_on_non_english` mark. Let me know

Reply:
I'm okay with `strict=False` for now then especially if there's a path to know how we can remove these in the future.

Reply:
> I'm okay with strict=False for now then especially if there's a path to know how we can remove these in the future.

im not clear on what this path is.  can you elaborate on this?

Reply:
> solve the issue (datetime parsing, separate topic)

From your investigation @smarie fixing the datetime parsing would allow us to remove the xfail?